blueprints/reducer/index.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path')
+
+module.exports = {
+  locals: function (options) {
+    // Return custom template variables here.
+    return {}
+  },
+
+  fileMapTokens: function (options) {
+    // Return custom tokens to be replaced in your files
+    return {
+      __token__: function (options) {
+        // logic to determine value goes here
+        return 'value'
+      }
+    }
+  },
+
+  // Should probably never need to be overriden
+
+  filesPath: function () {
+    return path.join(this.path, 'files')
+  },
+
+  beforeInstall: function (options) {},
+  afterInstall: function (options) {}
+}
```

Comment:
line 1 of blueprints/reducers/index.js.  You can have it just export an empty object since you don't need to override any of the hooks.

I think there may already be a `duck` blueprint that does something like this, but might be useful to have different versions of blueprints.